sql_generators/glean_usage/templates/event_monitoring_live_v1.view.sql

```diff
@@ -1,9 +1,11 @@
-CREATE MATERIALIZED VIEW
-IF
-  NOT EXISTS `{{ project_id }}.{{ derived_dataset }}.event_monitoring_live_v1`
-  OPTIONS
-    (enable_refresh = TRUE, refresh_interval_minutes = 60) AS
-    {% if dataset_id not in ["telemetry", "accounts_frontend", "accounts_backend"] %}
+CREATE OR REPLACE MATERIALIZED VIEW
+  `{{ project_id }}.{{ derived_dataset }}.event_monitoring_live_v1`
```

Comment:
IMO we should overwrite the materialized views every time so that any changes to their logic will get automatically deployed.  That does have the downside of losing the cached data when they're deployed, but since they're refreshing every hour anyway that doesn't seem like a dealbreaker to me, unless we decide to deploy views much more frequently.

Reply:
When we redeploy materialized views it will backfill the data every time. We do have a filter for `date(submission_timestamp)` for that reason as part of the query for when the materialized view gets initially deployed.
Also, materialized views cannot be replaced. They need to be deleted first: https://cloud.google.com/bigquery/docs/materialized-views-intro#limitations

Reply:
OK, I'll revert this part then.